### PR TITLE
feat: add UserInfoCard

### PR DESCRIPTION
**Pull request by chenbin92**

### CODE REVIEW

File: blocks/UserInfoCard/demo/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="/build/demo.css">
+</head>
+
+<body>
+  <div id="mountNode"></div>
+  <script type="text/javascript" src="//g.alicdn.com/third/react/15.3.2/react-with-addons.js"></script>
+  <script type="text/javascript" src="//g.alicdn.com/third/react/15.3.2/react-dom.js"></script>
+  <script src="/build/demo.js"></script>
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/demo/index.html]: prefer react16

g.alicdn.com/code/npm/??react/16.2.0/umd/react.development.js,react-dom/16.2.0/umd/react-dom.development.js

File: blocks/UserInfoCard/src/UserInfoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="user-info-card" style={styles.container}>
+        <Balloon
+          trigger={<a href="javascript:;">更多信息</a>}
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/src/UserInfoCard.jsx]: do not use javascript schema sentence

File: blocks/UserInfoCard/src/UserInfoCard.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          <div style={styles.content}>
+            <div style={styles.head}>
+              <img
+                src="https://img.alicdn.com/tfs/TB1nf.WjyqAXuNjy1XdXXaYcVXa-245-245.gif"
+                style={styles.avatar}
+              />
+              <div style={styles.baseInfo}>
+                <h5 style={styles.name}>张三</h5>
+                <p style={styles.deptName}>
+                  搜索事业部-工程效率-技术质量-质量系统平台
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/src/UserInfoCard.jsx]: 换一个不这么具体的

File: blocks/UserInfoCard/mock/api.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (yuanyan) [inline blocks/UserInfoCard/mock/api.json]: 遵循 mock 文件命名规则

File: blocks/UserInfoCard/package.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    "babel-eslint": "^8.0.3",
+    "eslint-plugin-react": "^7.5.1",
+    "react": "15.x",
+    "react-copy-to-clipboard": "^4.2.3",
+    "react-dom": "15.x"
+  },
+  "blockConfig": {
+    "name": "user-info-card",
+    "description": "",
+    "chinese": "用户信息展示",
```
Comment (yuanyan) [inline blocks/UserInfoCard/package.json]: chinese 是什么意思？

File: blocks/UserInfoCard/package.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    "@icedesign/skin": "latest",
+    "babel-eslint": "^8.0.3",
+    "eslint-plugin-react": "^7.5.1",
+    "react": "15.x",
+    "react-copy-to-clipboard": "^4.2.3",
+    "react-dom": "15.x"
+  },
+  "blockConfig": {
+    "name": "user-info-card",
+    "description": "",
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: dup

File: blocks/UserInfoCard/package.json (hunk 5 above)
Comment (wssgcg1213) [reply blocks/UserInfoCard/package.json]: rename to `title`

File: blocks/UserInfoCard/package.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    "chinese": "用户信息展示",
+    "category": "info-display",
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: consider renaming to categories[]

File: blocks/UserInfoCard/package.json [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    "@icedesign/demo-layout": "^0.1.2",
+    "prop-types": "^15.5.8"
+  },
+  "devDependencies": {
+    "@icedesign/skin": "latest",
+    "babel-eslint": "^8.0.3",
+    "eslint-plugin-react": "^7.5.1",
+    "react": "15.x",
+    "react-copy-to-clipboard": "^4.2.3",
+    "react-dom": "15.x"
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: 需要兼容16

File: blocks/UserInfoCard/package.json [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+  "dependencies": {
+    "@icedesign/base": "^0.1.0",
+    "@icedesign/card": "^0.1.1",
+    "@icedesign/demo-layout": "^0.1.2",
+    "prop-types": "^15.5.8"
+  },
+  "devDependencies": {
+    "@icedesign/skin": "latest",
+    "babel-eslint": "^8.0.3",
+    "eslint-plugin-react": "^7.5.1",
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: del

File: blocks/UserInfoCard/package.json [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "blockConfig": {
+    "name": "user-info-card",
+    "description": "",
+    "chinese": "用户信息展示",
+    "category": "info-display",
+    "snapshot":
+      "https://img.alicdn.com/tfs/TB1NxO4l8fH8KJjy1XbXXbLdXXa-978-668.png",
+    "sketchURL": "",
+    "icelandURL": "",
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: if no value, del it

File: blocks/UserInfoCard/package.json [hunk 11]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@icedesign/user-info-card",
```
Comment (wssgcg1213) [inline blocks/UserInfoCard/package.json]: -block

**Pull request merged**